bluemira/optimisation/_geometry/optimise.py

```diff
@@ -63,28 +63,28 @@ def optimise_geometry(
 
     Parameters
     ----------
-    geom: GeometryParameterisation
+    geom
```

Comment:
Unfortunately as of #2192 we are have colons...
```suggestion
    geom: 
```